src/applications/facility-locator/actions/index.js

```diff
@@ -323,74 +323,63 @@ export const genBBoxFromAddress = query => {
  * from the coordinates center of the map
  */
 export const genSearchAreaFromCenter = query => {
-  const { lat, lng } = query;
+  const { lat, lng, currentMapBoundsDistance, currentBounds } = query;
   return dispatch => {
-    const types = TypeList;
-    mbxClient
-      .reverseGeocode({
-        countries: CountriesList,
-        types,
-        query: [lng, lat],
-      })
-      .send()
-      .then(({ body: { features } }) => {
-        const coordinates = features[0].center;
-        const zip =
-          features[0].context.find(v => v.id.includes('postcode')) || {};
-        const location = zip.text || features[0].place_name;
-        const featureBox = features[0].box;
-
-        const sw = coordinates[0] - BOUNDING_RADIUS;
-        const se = coordinates[1] - BOUNDING_RADIUS;
-        const nw = coordinates[0] + BOUNDING_RADIUS;
-        const ne = coordinates[1] + BOUNDING_RADIUS;
-        let minBounds = [sw, se, nw, ne];
-        if (featureBox) {
-          minBounds = [
-            Math.min(featureBox[0], sw),
-            Math.min(featureBox[1], se),
-            Math.max(featureBox[2], nw),
-            Math.max(featureBox[3], ne),
-          ];
-        }
+    if (currentMapBoundsDistance > 500) {
```

Comment:
Let's put **500** into a const in src/applications/facility-locator/constants/index.js.